go.mod

```diff
@@ -1,12 +1,19 @@
 module github.com/liquidmetal-dev/fl
 
-go 1.22
+go 1.23
+
+toolchain go1.23.4
+
+replace (
+	github.com/liquidmetal-dev/flintlock/api => ../flintlock/api
```

Comment:
These 2 replacments are temporary and will be removed after the change has been released in flintlock.